### PR TITLE
[HDInsight on AKS] Api version 2023-11-1-preview support

### PR DESCRIPTION
**Pull request by yuc-Li**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by yuc-Li** [comment]
Hi @m-nash, I got these errors in pipeline, but I didn't find these issues in the file.
image
and I ran cspell locally without any report. Could you help me with this?
image

File: sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/src/Azure.ResourceManager.HDInsight.Containers.csproj [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Version>1.0.0-beta.3</Version>
```
Comment (ArcturusZhang) [inline sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/src/Azure.ResourceManager.HDInsight.Containers.csproj:3]: beta.3 is an unreleased version already, we do not need to bump it. Please revert this change.

File: sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,18 @@
...
+- Cluster Pool
+    - Enable create cluster pool with user network profile.
+    - Enable get cluster pool available upgrade versions.
+- Cluster
+    - Enable create Ranger cluster.
+    - Enable get cluster available upgrade versions.
+    - Enable set internal ingress.
+    - Enable check if the cluster name is available.
+    - Enable upgrade hot fix for cluster.
+    - Enable upgrade node os for cluster.
```
Comment (ArcturusZhang) [inline sdk/hdinsight/Azure.ResourceManager.HDInsight.Containers/CHANGELOG.md:15]: Please add your changelog to the latest unreleased version (aka 1.0.0-beta.3) as a new item

**Comment by yuc-Li** [comment]
> Hi @m-nash, I got these errors in pipeline, but I didn't find these issues in the file. image and I ran cspell locally without any report. Could you help me with this? image

Resolved.

**Comment by yuc-Li** [comment]
Do not merge temporarily, thanks.

**Review by ArthurMa1978 — APPROVED** [review]

**Comment by yuc-Li** [comment]
Hello~ @ArthurMa1978, I want to figure out a process problem. Will the code automatically release package or manually release after merging?  This will be helpful to our future plans. Thanks~

**Pull request merged**